Rules/CommonScripts/KickAFK.as

```diff
@@ -75,7 +75,17 @@ void onTick(CRules@ this)
 		}
 		else
 		{
-			p.client_ChangeTeam(this.getSpectatorTeamNum());
+			if(getRules().gamemode_name=="CTF"||getRules().gamemode_name=="TTH")
+			{
+				p.client_ChangeTeam(this.getSpectatorTeamNum());
+			}
+			else
+			{
+				client_AddToChat("You have been AFK way too long, this server may not have a spectator team to swap you to!",SColor(255,0,0,0));
+				client_AddToChat("[!] Please either:",SColor(255,0,0,0));
+				client_AddToChat("-Type !m in chat while being AFK, re-type it when you're back.",SColor(255,0,0,0));
```

Comment:
I wouldn't call this a 'fix'. You're just telling the player to use a totally unrelated feature to fix this issue. The message it posts is useless since the player won't even see the message until they're back from being AFK. The message also takes up five chat lines for something that only needs to take up two at most.